ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/DiffExtensions.scala

```diff
@@ -3,51 +3,45 @@
 
 package com.daml.ledger.api.testtool.infrastructure
 
-import ai.x.diff.{Comparison, DiffShow, Different, Identical, green, red}
+import com.softwaremill.diffx._
 import scalaz.{@@, Tag}
 
 trait DiffExtensions {
-  implicit def diffShowTag[A, T](implicit diffShowA: DiffShow[A]): DiffShow[A @@ T] =
-    new DiffShow[A @@ T] {
-      override def show(t: A @@ T): String = diffShowA.show(Tag.unwrap(t))
 
-      override def diff(left: A @@ T, right: A @@ T): Comparison =
-        diffShowA.diff(Tag.unwrap(left), Tag.unwrap(right))
-    }
+  implicit def diffTag[A, T](implicit diffA: Diff[A]): Diff[A @@ T] = {
+    new Diff[A @@ T] {
+      override def apply(left: A @@ T, right: A @@ T, toIgnore: List[FieldPath]): DiffResult = {
+        diffA.apply(Tag.unwrap(left), Tag.unwrap(right))
+      }
 
-  implicit def diffShowSeq[T](implicit diffShowT: DiffShow[T]): DiffShow[Seq[T]] =
-    new DiffShow[Seq[T]] {
-      override def show(t: Seq[T]): String = t.toString
+    }
+  }
 
-      override def diff(left: Seq[T], right: Seq[T]): Comparison = {
+  implicit def diffSeq[T](implicit diffT: Diff[T]): Diff[Seq[T]] =
```

Comment:
So the new library doesn't have a native `Diff[Seq[T]]` either?

Reply:
Good catch! There's already available instance of `Diff[Seq[T]]`, so I've removed the custom one.